cli/src/athenapdf.js

```diff
@@ -110,8 +111,16 @@ const loadOpts = {
     "extraHeaders": athena.cache ? "" : "pragma: no-cache\n"
 };
 
+// Enum for Electron's marginType codes
+var MarginEnum = {
```

Comment:
Small thing: can we use a `const` here instead (since it should read-only) ? Thanks 👍